### ISSUE TITLE
Header "Host" fuzzing isn't working correctly

### ISSUE THREAD

**Issue by allyshka** [xmendez/wfuzz#31]
I was try to bruteforce server's virtual hosts and found that "Host" parameter is doesn't affect fuzzing.
`wfuzz.py -H "Host: FUZZ.test.local" -w wordlist/general/test.txt http://127.0.0.1 -sc 200`

After check source codes I find out that **reqresp** lib is rewrite fuzzed _host_ parameter of request inside **setUrl** method.
- **framework\fuzzer\fuzzobjects.py:210**

```
    newreq.setUrl(rawUrl)
```
- **externals\reqresp\Request.py:138**

```
        self._headers["Host"]=self.__host
```

I used this fast hack to complete my task. Replace line at **externals\reqresp\Request.py:138**
from:

```
        self._headers["Host"]=self.__host
```

to

```
        if "Host" not in self._headers:
            self._headers["Host"]=self.__host
```

But, isn't good to change logic of external library.
I guess, correct solution is add compiled headers after **setUrl** method here **framework\fuzzer\fuzzobjects.py:210** if they involved in fuzzing process.

**Comment by darkbyte-ru** [comment]
Strange way to generate HTTP request. 

If you build plain text HTTP request, why not to add URI to it by your self?

In addition to @allyshka problem, this cause problem with fuzz "http://FUZZ/" URI, because in from_seed method regexp check look only in rawReq, but FUZZ was in completeUrl.

**Comment by xmendez** [comment]
Fixed in new 2.2.0 release.
